pkg/ui/workspaces/crdb-js-proto/BUILD.bazel

```diff
@@ -0,0 +1,82 @@
+load("@aspect_rules_js//js:defs.bzl", "js_binary", "js_run_binary", "js_test")
+load("@aspect_rules_js//npm:defs.bzl", "npm_package")
+load("@aspect_rules_ts//ts:defs.bzl", "ts_project")
+load("@bazel_skylib//rules:write_file.bzl", "write_file")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("@npm//:defs.bzl", "npm_link_all_packages")
+load("//docs/generated/http:defs.bzl", "PROTOBUF_TARGETS")
+load("//pkg/ui/workspaces/db-console/src/js:defs.bzl", "proto_sources")
+
+npm_link_all_packages(name = "node_modules")
+
+genrule(
+    name = "protos",
+    outs = ["dist"],
+    cmd = """ \
+    mkdir -p $(OUTS); \
+    DESCRIPTOR_SET_IN=$$(for word in $(SRCS); do echo $$word; done | grep '\\.bin$$' | tr -s '\\n' ':'); \
+    PROTO_PATHS=$$(for f in $(locations :proto_sources); \
+        do if [[ "$$f" =~ ^.*/pkg/.*/_virtual_imports/.*_proto/.*.proto$$ ]]; \
+            then echo $$f | sed 's|.*pkg/.*/_virtual_imports/.*_proto/||' | sed 's|.bin$$||'; \
+        fi; done | sort -u); \
+    export PATH=$$PATH:$$(dirname $(NODE_PATH)); \
```

Comment:
What is this `NODE_PATH`? Seems undefined.

Reply:
It is necessary to find `node.js` executable. `protoc-gen-es` plugin relies on `node` executable and assumes it should be available on the `PATH` which is not true in our case. That's why `@nodejs_toolchains//:resolved_toolchain` is added to enable `node.js` which can be found on `NODE_PATH` path.